guides/common/modules/con_prerequisites-for-using-discovery.adoc

```diff
@@ -3,6 +3,11 @@
 [id="prerequisites-for-using-discovery_{context}"]
 = Prerequisites for using Discovery
 
+[role="_abstract"]
+Before you start using Discovery, ensure that your environment meets the following requirements.
+
+The requirements for network-boot provisioning include:
```

Comment:
```suggestion
```

I think we can drop this line. IMO title and introduction are enough. Also, the wording was off. network-boot IMO is not equal to Host Discovery.